### PR TITLE
Update notebooks.md

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: python-data-analysis/python-data-analysis-core/analysis-environments/notebooks.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ You and anyone you've shared your notebook with can also comment on any portion
 
 They also work like *extended versions* of scripts.
 
-Each piece of code can be run and tested independently from other code. However, you can also run all of the code at once.
+Each piece of code can be run and tested independently from other code. Y can also run all of the code at once.
```
Comment (Stefan-Stojanovic) [inline python-data-analysis/python-data-analysis-core/analysis-environments/notebooks.md:36]:
```suggestion
Each piece of code can be run and tested independently from other code. You can also run all of the code at once.
```

**Review by Stefan-Stojanovic — CHANGES_REQUESTED** [review]

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| notebooks | No matching footnotes found. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/python-data-analysis/python-data-analysis-core/analysis-environments/notebooks.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/python-data-analysis/python-data-analysis-core/analysis-environments/notebooks.md?ref=a60c10a5ff5f7a19ea7e0f73fc2cab4620af5355)

**Review by Stefan-Stojanovic — APPROVED** [review]

**Pull request merged**